src/sentry/models/rollbackuserdata.py

```diff
@@ -0,0 +1,32 @@
+from uuid import uuid4
+
+from django.db import models
+from django.db.models import UniqueConstraint
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models.base import DefaultFieldsModel, region_silo_model
+from sentry.db.models.fields.foreignkey import FlexibleForeignKey
+from sentry.db.models.fields.hybrid_cloud_foreign_key import HybridCloudForeignKey
+
+
+@region_silo_model
+class RollbackUser(DefaultFieldsModel):
+    """
+    Stores a summary of every organization member's year-in-review information to power the 2024 Sentry Rollback.
+    """
+
+    __relocation_scope__ = RelocationScope.Excluded
+
+    user_id = HybridCloudForeignKey("sentry.User", on_delete="CASCADE")
+    organization = FlexibleForeignKey("sentry.Organization")
+    uuid = models.UUIDField(default=uuid4, editable=False, db_index=True)
+    share_uuid = models.UUIDField(default=uuid4, editable=False, db_index=True)
```

Comment:
Did you mean to remove the unique here rather than the db_index?

Reply:
Tup this is a mistake, I read your comment backwards and thought that db_index enforced uniqueness, not the other way around. Updating this now